ios/FluentUI/Core/ControlHostingView.swift

```diff
@@ -36,17 +36,10 @@ open class ControlHostingView: UIView {
     ///
     /// - Parameter controlView: An `AnyView`-wrapped component to host.
     init(_ controlView: AnyView) {
-        self.controlView = controlView
+        hostingController = FluentUIHostingController.init(rootView: AnyView(controlView))
```

Comment:
Is the `controlView` not already wrapped in an `AnyView`?

Reply:
Good catch!